src/hct_mis_api/api/endpoints/lookups/base.py

```diff
@@ -25,9 +26,12 @@ def get(self, request: "Request", format: Optional[Any] = None) -> Response:
         return Response(dict(IDENTIFICATION_TYPE_CHOICE))
 
 
-class Country(HOPEAPIView):
+class CountryAPIView(HOPEAPIView):
+    serializer_class = CountrySerializer
+
     def get(self, request: "Request", format: Optional[Any] = None) -> Response:
-        return Response(dict(Countries()))
+        serializer = self.serializer_class(Country.objects.all(), many=True)
```

Comment:
let's change this in a viewset @pkujawa

Reply:
@domdinicola but we only want to retrieve list, right? All the lookups in this file are constructed this way